backend/src/packages/articles/article.repository.ts

```diff
@@ -91,6 +92,28 @@ class ArticleRepository implements IRepository {
     return ArticleEntity.initialize(article);
   }
 
+  public async getUserActivity({
+    userId,
+    activityFrom,
+    activityTo,
+  }: {
+    userId: number;
+    activityFrom: string;
+    activityTo: string;
+  }): Promise<UserActivityResponseDto[]> {
+    const userActivity = (await this.articleModel
+      .query()
+      .select(
+        this.articleModel.raw('date(created_at), date(updated_at) as date'),
+        this.articleModel.raw('count(*)'),
+      )
+      .where({ userId })
+      .whereBetween('createdAt', [activityFrom, activityTo])
+      .groupByRaw('date(created_at), date(updated_at)')) as unknown;
```

Comment:
```suggestion
    const userActivity = await this.articleModel
      .query()
      .select(
        this.articleModel.raw('date(created_at), date(updated_at) as date'),
        this.articleModel.raw('count(*)'),
      )
      .where({ userId })
      .whereBetween('createdAt', [activityFrom, activityTo])
      .groupByRaw('date(created_at), date(updated_at)')
      .castTo<UserActivityResponseDto[]>();
```

Reply:
I didn't quite understand you, could you please clarify what you mean

Reply:
I mean not using type cast to unknown then to needed type
And just declare resulting type in `.castTo`

Reply:
I understood the part about unknown, but I didn't understand what is the method castTo, it's not present in knex or objection, where should I define it?